src/main/resources/assets/computercraft/lua/rom/programs/shell.lua

```diff
@@ -104,9 +104,9 @@ end
 -- Colours
 local promptColour, textColour, bgColour
 if term.isColour() then
-    promptColour = colours.yellow
-    textColour = colours.white
-    bgColour = colours.black
+    promptColour = colors[ settings.get( "shell.prompt_color" ) ]
```

Comment:
Now that I think about it there should be some fallback here. Cause if this setting is incorrect shell crashes and whole computer is unusable. I propose `promptColour = colors[ settings.get( "shell.prompt_color" ) ] or colours.yellow`. It would be nice to secure all stuff that takes color from settings like this but a shell is a must.